xarray/core/variable.py

```diff
@@ -1318,12 +1327,13 @@ def chunk(self, chunks=None, name=None, lock=False):
         return self.copy(deep=False)
 
     def __getitem__(self, key):
-        key = self._item_key_to_tuple(key)
-        values = self._indexable_data[key]
+        dims, index_tuple = self._broadcast_indexes(key)
+        assert len(dims) <= 1
```

Comment:
Rather than add this `assert`, can we change the condition in the `if` statement below? I think we can simply make it `if len(dims) != 1:`.